projects/1.18/assets/andycats-flowers/andycats_flowers/lang/zh_cn.json

```
@@ -0,0 +1,141 @@
+{
+  "block.andycats_flowers.cyan_cosmos": "青色大波斯菊",
+  "block.andycats_flowers.purple_gladiolus": "紫色剑兰",
+  "block.andycats_flowers.tall_cyan_gladiolus": "高青色剑兰",
+  "block.andycats_flowers.pink_mallow": "粉红色锦葵",
+  "block.andycats_flowers.tall_white_daffodil": "高白色水仙",
+  "block.andycats_flowers.cyan_echinacea": "青色锥菊",
+  "block.andycats_flowers.tall_red_cosmos": "高红色大波斯菊",
+  "block.andycats_flowers.tall_purple_iris": "高紫色鸢尾",
+  "block.andycats_flowers.tall_purple_echinacea": "高紫色锥菊",
+  "block.andycats_flowers.light_blue_commelina": "淡蓝色鸭跖草",
+  "block.andycats_flowers.purple_iris": "紫色鸢尾",
+  "block.andycats_flowers.white_echinacea": "白色锥菊",
+  "block.andycats_flowers.tall_maroon_mallow": "高褐红色锦葵",
+  "block.andycats_flowers.tall_white_gladiolus": "高白色剑兰",
+  "block.andycats_flowers.blue_cowslip": "蓝色樱草",
+  "block.andycats_flowers.light_blue_cosmos": "淡蓝色大波斯菊",
+  "block.andycats_flowers.cyan_gladiolus": "青色剑兰",
+  "block.andycats_flowers.blue_iris": "蓝色鸢尾",
+  "block.andycats_flowers.tall_cyan_echinacea": "高青色锥菊",
+  "block.andycats_flowers.mixed_cosmos": "杂色大波斯菊",
+  "block.andycats_flowers.tall_purple_gladiolus": "高紫色剑兰",
+  "block.andycats_flowers.white_gladiolus": "白色剑兰",
+  "block.andycats_flowers.blue_cosmos": "蓝色大波斯菊",
+  "block.andycats_flowers.tall_pink_echinacea": "高粉红色锥菊",
+  "block.andycats_flowers.tall_orange_calla_lily": "高橙色马蹄莲",
+  "block.andycats_flowers.pink_calla_lily": "粉红色马蹄莲",
+  "block.andycats_flowers.blue_echinacea": "蓝色锥菊",
+  "block.andycats_flowers.yellow_daffodil": "黄色水仙",
+  "block.andycats_flowers.cyan_calla_lily": "青色马蹄莲",
+  "block.andycats_flowers.tall_cyan_calla_lily": "高青色马蹄莲",
+  "block.andycats_flowers.purple_echinacea": "紫色锥菊",
+  "block.andycats_flowers.tall_black_calla_lily": "高黑色马蹄莲",
+  "block.andycats_flowers.tall_yellow_daffodil": "高黄色水仙",
+  "block.andycats_flowers.red_echinacea": "红色锥菊",
+  "block.andycats_flowers.tall_light_blue_cosmos": "高淡蓝色大波斯菊",
+  "block.andycats_flowers.tall_blue_gladiolus": "高蓝色剑兰",
+  "block.andycats_flowers.tall_cyan_cosmos": "高青色大波斯菊",
+  "block.andycats_flowers.tall_purple_cosmos": "高紫色大波斯菊",
+  "block.andycats_flowers.tall_red_commelina": "高红色鸭跖草",
+  "block.andycats_flowers.tall_magenta_echinacea": "高品红色锥菊",
+  "block.andycats_flowers.tall_mixed_cosmos": "高杂色大波斯菊",
+  "block.andycats_flowers.tall_light_blue_echinacea": "高淡蓝色锥菊",
+  "block.andycats_flowers.tall_yellow_cowslip": "高黄色樱草",
+  "block.andycats_flowers.white_baneberry": "白色毒野莓",
```

```suggestion
  "block.andycats_flowers.white_baneberry": "白色升麻果",
```
http://www.iplant.cn/info/%E7%B1%BB%E5%8F%B6%E5%8D%87%E9%BA%BB%E5%B1%9E
baneberry作为俗名倒确实是`毒莓`的意思，不知道中文里有没有这种物种的俗名

---

这里这里